tasks/task1.js

```diff
@@ -0,0 +1,44 @@
+//what if numbers are equal
+
+function range1(x, y){
+
+ if(!Number.isInteger(x) || !Number.isInteger(y)){
+   return undefined;
+ } else if (x === y) {
+   return [x];
+ }
+
+ let newArr = [];
```

Comment:
Nitpick: this variable could be a `const` as you are not reassigning it.  
Also, variable name shortening is useful when you could get a tangibly shorter name, in this case, either `newArray` or `result` would be better, IMHO. 😉